raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -323,3 +328,116 @@ def test_mediated_transfer_calls_pfs(raiden_network, token_addresses):
         )
         app0.raiden.mediate_mediated_transfer(transfer)
         assert patched.call_count == 2
+
+
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('number_of_nodes', [4])
+def test_mediated_transfer_with_allocated_fee(
```

Comment:
Test Request: This test is nice but I would also like to see a test where as we discussed in the standup, the transfer fails, since a mediator took more fees than he should have and the final amount reaching the target is less than the initially intended transfer amount.

Reply:
Will add the requested test

Reply:
:+1: